app/views/users/show.html.erb

```diff
@@ -4,6 +4,8 @@
   </div>
 </div>
 
+
+<% if current_user ==@user || !@user.private? || current_user.leaders.include?(@user) %>
```

Comment:
make sure to add in a space between the `==` and `@user`, just to be consistent with syntax